cmd/lotus-miner/proving.go

```diff
@@ -304,7 +304,7 @@ var provingRecoverFaultsCmd = &cli.Command{
 		},
 	},
 	Action: func(cctx *cli.Context) error {
-		if cctx.NArg() < 1 {
+		if cctx.NArg() != 1 {
```

Comment:
no, this one iterates over `cctx.Args().Slice()` so it can take more than 1 arg too, undo this